README.md

```diff
@@ -1,24 +1,24 @@
 
-An architecture developed and revised by the [FilledStacks](https://www.youtube.com/filledstacks) community. This architecture was initially a version of Mvvm as [described in this video](https://youtu.be/kDEflMYTFlk). Since then Filledstacks app development team has built 6 production applications with various requirements. This experience along with countless requests for improvements and common functionality is what sparked the creation of this architecture package. It aims to provide **common functionalities to make app development easier** as well as code principles to use during development to ensure your code stays maintainable.
+An architecture developed and revised by the [FilledStacks](https://www.youtube.com/filledstacks) community. This architecture was initially a version of MVVM as [described in this video](https://youtu.be/kDEflMYTFlk). Since then Filledstacks app development team has built 6 production applications with various requirements. This experience along with countless requests for improvements and common functionality is what sparked the creation of this architecture package. It aims to provide **common functionalities to make app development easier** as well as code principles to use during development to ensure your code stays maintainable.
 
-**If you're Reading this disclaimer the series that does a deepdive on this architecture has not been published yet.**
+**If you're Reading this disclaimer the series that does a deep-dive on this architecture has not been published yet.**
 
 ## How Does it work
 
 The architecture is very simple. It consists of 3 major pieces, everything else is up to your implementation style. These pieces are:
 
-- **View**: Shows the UI to the user. Single widgets also qualifies as views (for consistency in terminology) a view in this case is not a "Page" it's just a UI representation.
-- **ViewModel**: Manages the state of the View, business logic and any other logic as required from user interaction. It does this by making use of the services
-- **Services**: A wrapper of a single functionality / feature set. This is commonly used to wrap things like showing a dialog, wrapping database functionality, integrating an api, etc.
+- **View**: Shows the UI to the user. Single widgets also qualify as views (for consistency in terminology) a view, in this case, is not a "Page" it's just a UI representation.
+- **ViewModel**: Manages the state of the View, business logic, and any other logic as required from user interaction. It does this by making use of the services
+- **Services**: A wrapper of a single functionality/feature set. This is commonly used to wrap things like showing a dialog, wrapping database functionality, integrating an API, etc.
 
-Lets go over some of those principles to follow during development.
+Let's go over some of those principles to follow during development.
 
-- Views should never MAKE USE of a service directly.
+- Views should never MAKE USE a of service directly.
```

Comment:
This sentence doesn't make sense. Change back to 

Views should never MAKE USE of a service directly. 'a of' doesn't bear meaning in english

Reply:
yea I see 😂🤣